rust/BUILD.md

```diff
@@ -163,6 +173,23 @@ cargo build --package benchmarks --profile profiling
 ```
 run `./target/profiling/benchmarks` with Intel VTune 
 
+### DHAT
```

Comment:
you are missing some info here:
what is dhat? what does it do? 
why would i want to learn to use it?
and finally, a link to some reference would be nice to have